### DISCUSSION TITLE
Why am I getting the "Account Activation Pending.." issue after recreating the docker container?

### DISCUSSION THREAD

**Discussion by Tony20221** [open-webui/open-webui#5930] [Q&A]
I am on Windows 11 and created the openwebui using this command a couple of weeks ago:
`docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

I couldn't log in so I recreated the docker container and now I get the Account Activation Pending.. message.
The FAQ says to reset the admin password using htpasswd -bnBC 10 "" your-new-password | tr -d ':\n'

I don't have a htpasswd file. Are the instructions for Linux?
Execute the command where?

Anyway how is openwebui complaining about reused email addresses and Account Activation Pending issues after recreating the container? Deleting the container is supposed to delete everything in it. It's as if this info is saved somewhere outside the container.

How do I start completely fresh?

**Comment by Mr-HatnBootz** [comment]
I have managed to solve this on my end using Windows 10 docker desktop / powershell .  I used the same command as described above.

The htpasswd is not a file, it is a command that is a part of the apache2-utils package.

in order to update the settings in docker use the following command in powershell "docker exec -it open-webui /bin/bash"

once you see root followed by some numbers and /app/backend#  then update the repo's  by "apt-get update" 

Then install sqlite3 and apache2-utils. with the following "apt-get install sqlite3 apache2-utils"

Now with apache2-utils installed you can use the htpasswd above to generate your password. 

change into the data directory "cd data" and use sqlite on webui.db   "sqlite3 webui.db"  the shell will change, can use .tables to show all the tables.  

then use " SELECT * FROM auth; " to show all the current users.  then issue the following command to update the password "UPDATE auth SET password='HASH' WHERE email='email account here'; "   once done you can type .exit and close out powershell. 

Then you should be able to login. 

*Late edit - I have created a video guide on how to do this on youtube, just search for "Ollama Open-Webui Password reset."
